pettingzoo/test/api_test.py

```diff
@@ -26,75 +23,39 @@ def test_observation(observation, observation_0):
     else:
         warnings.warn("Observation is not NumPy array")
 
 
 def test_observation_action_spaces(env, agent_0):
     assert isinstance(env.action_spaces, dict), "action_spaces must be a dict"
     assert isinstance(env.observation_spaces, dict), "observation_spaces must be a dict"
-    assert (
-        len(env.observation_spaces) == len(env.action_spaces) == len(env.agents)
-    ), "observation_spaces, action_spaces, and agents must have the same length"
+    assert len(env.observation_spaces) == len(env.action_spaces) == len(env.agents), "observation_spaces, action_spaces, and agents must have the same length"
 
     for agent in env.agents:
-        assert isinstance(
-            env.observation_spaces[agent], gym.spaces.Space
-        ), "Observation space for each agent must extend gym.spaces.Space"
-        assert isinstance(
-            env.action_spaces[agent], gym.spaces.Space
-        ), "Agent space for each agent must extend gym.spaces.Space"
-        if not (
-            isinstance(env.observation_spaces[agent], gym.spaces.Box)
-            or isinstance(env.observation_spaces[agent], gym.spaces.Discrete)
-        ):
-            warnings.warn(
-                "Observation space for each agent probably should be gym.spaces.box or gym.spaces.discrete"
-            )
-        if not (
-            isinstance(env.action_spaces[agent], gym.spaces.Box)
-            or isinstance(env.action_spaces[agent], gym.spaces.Discrete)
-        ):
-            warnings.warn(
-                "Action space for each agent probably should be gym.spaces.box or gym.spaces.discrete"
-            )
-        if (not isinstance(agent, str)) and agent != "env":
-            warnings.warn(
-                "Agent's are recommended to have numbered string names, like player_0"
-            )
-        if not isinstance(agent, str) or not re.match(
-            "[a-z]+_[0-9]+", agent
-        ):  # regex for ending in _<integer>
-            warnings.warn(
-                'We recommend agents to be named in the format <descriptor>_<number>, like "player_0"'
-            )
-        if not isinstance(
-            env.observation_spaces[agent], env.observation_spaces[agent_0].__class__
-        ):
-            warnings.warn(
-                "The class of observation spaces is different between two agents"
-            )
-        if not isinstance(
-            env.action_spaces[agent], env.action_spaces[agent_0].__class__
-        ):
+        assert isinstance(env.observation_spaces[agent], gym.spaces.Space), "Observation space for each agent must extend gym.spaces.Space"
+        assert isinstance(env.action_spaces[agent], gym.spaces.Space), "Agent space for each agent must extend gym.spaces.Space"
+        if not (isinstance(env.observation_spaces[agent], gym.spaces.Box) or isinstance(env.observation_spaces[agent], gym.spaces.Discrete)):
```

Comment:
The second parameter of `isinstance()` can be a tuple.
* https://docs.python.org/3/library/functions.html#isinstance
```suggestion
        if not isinstance(env.observation_spaces[agent], (gym.spaces.Box, gym.spaces.Discrete)):
```